demo/OpenVINO/cpp/yolox_openvino.cpp

```diff
@@ -41,6 +42,8 @@ void blobFromImage(cv::Mat& img, Blob::Ptr& blob){
     int channels = 3;
     int img_h = img.rows;
     int img_w = img.cols;
+    std::vector<float> mean = {0.485, 0.456, 0.406};
+    std::vector<float> std = {0.229, 0.224, 0.225};
```

Comment:
We remove the normalization in the new version, so you may have to pull our latest version

Reply:
I have removed the normalization in my code. Thank you.